packages/test/local-server-tests/src/test/connectionMode.spec.ts

```diff
@@ -79,19 +79,20 @@ describe("Logging Last Connection Mode ", () => {
 		const urlResolver = new LocalResolver();
 		const codeLoader = new LocalCodeLoader([[codeDetails, runtimeFactory]]);
 
-		const loader = new Loader({
+		const loaderProps: ILoaderProps = {
 			urlResolver,
 			documentServiceFactory,
 			codeLoader,
 			logger,
-		});
-		loaderContainerTracker.add(loader);
+		};
 
-		return createAndAttachContainer(
+		const container1 = await createAndAttachContainerUsingProps(
```

Comment:
Wondering if this has any impact on `loaderContainerTracker` functionality. Right now we add the loader and then create+attach the container. The new code creates and attaches it, and only after adds it to the tracker. We should make sure that doesn't break any of its functionality.